Syntax error "Cannot find module './src/interfaces'."

When using the typescript definitions, it gives an error "Cannot find module './src/interfaces'."

as "src/interfaces" does not exist in the dist folder,

version 0.2.1, same error for 0.2.0

![image](https://user-images.githubusercontent.com/5888506/46254833-b7587900-c495-11e8-9359-a3d646144a66.png)

--- a/package.json
+++ b/package.json
@@ -6,7 +6,7 @@
   "types": "dist/index.d.ts",
   "scripts": {
     "build": "npm run clean; tsc; npm run copy-assets",
-    "copy-assets": "cp src/worker.js dist; cp index.d.ts dist",
+    "copy-assets": "cp src/worker.js dist; cp src/index.d.ts dist",
     "clean": "rm -rf dist",
     "pretest": "npm run lint",
     "test": "node --experimental-worker node_modules/.bin/jest",

--- a/src/index.d.ts
+++ b/src/index.d.ts
@@ -1,4 +1,4 @@
-import {Config} from './src/interfaces'
+import { Config } from './interfaces'
 
 export function job(handler: Function, config?: Config): Promise<any>
 export function stop(): void

